www/js/lib/uiUtil.js

```diff
@@ -244,6 +252,7 @@ define([], function() {
         displayActiveContentWarning: displayActiveContentWarning,
         displayFileDownloadAlert: displayFileDownloadAlert,
         isElementInView: isElementInView,
-        htmlEscapeChars : htmlEscapeChars
+        htmlEscapeChars : htmlEscapeChars,
```

Comment:
@vasusharma7 Remove the extra space before the colon, like lines 248-254. And in the next line below.

Reply:
Ok sure